<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMainWindow</class>
 <widget class="QDialog" name="QMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_write">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background: qlineargradient(spread:pad, x1:0.148, y1:1, x2:1, y2:1, stop:0.233503 rgba(53, 132, 228, 255), stop:1 rgba(26, 95, 180, 255));
	color: white;
	border-top-left-radius: 8px;
	border-bottom-right-radius: 8px;
	qproperty-cursor: pointingHandCursor;
}

QPushButton:hover {
	background: qlineargradient(spread:pad, x1:0.148, y1:1, x2:1, y2:1, stop:0.30102 rgba(0, 115, 255, 255), stop:1 rgba(12, 45, 85, 255));
}</string>
   </property>
   <property name="text">
    <string>Write</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Tahoma</family>
     <pointsize>30</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Remark</string>
   </property>
   <property name="textFormat">
    <enum>Qt::PlainText</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>161</width>
     <height>61</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>161</width>
     <height>61</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	border: 4px solid rgb(53, 132, 228);
	border-top-left-radius: 8px;
	border-bottom-right-radius: 8px;
	qproperty-cursor: pointingHandCursor;
}

QPushButton:hover {
	border: 6px solid rgb(26, 95, 180);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="et_remark">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto Slab</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border-radius: 8px;
	border: 3px solid rgb(53, 132, 228);
	padding-left: 7px;
}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Normal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
